book_source/06_reference/02_models/biocro.Rmd

```diff
@@ -1,31 +1,39 @@
 ## BioCro {#models-biocro}
 
-| Model Information ||
-| -- | -- |
-| Home Page | |
-| Source Code | |
-| License |  |
-| Authors | |
-| PEcAn Integration | |
+| Model Information |                                                                   |
+| --                | --                                                                |
+| Home Page         | http://biocro.r-forge.r-project.org/                              |
+| Source Code       | https://github.com/ebimodeling/biocro                             |
+| License           |                                                                   |
+| Authors           | Fernando E. Miguez, Deepak Jaiswal, Justin McGrath, David LeBauer |
+| PEcAn Integration | David LeBauer, Dan Wang                                           |
 
-**Introduction**
+### Introduction
 
-Introduction about model
+BioCro is a model that estimates photosynthesis at the leaf, canopy, and ecosystem levels and determines plant biomass allocation and crop yields, using underlying physiological and ecological processes to do so. 
 
-**PEcAn configuration file additions**
+### PEcAn configuration file additions
 
-Should list the model specific additions to the PEcAn file here
+The following sections of the PEcAn XML are relevant to the BioCro model: 
 
-**Model specific input files**
+- `model`
+  - `revision` -- Model version number
+- `run`
+  - `site/id` -- ID associated with desired site from BETYdb site entry
+  - `inputs`
+    - `met/output` -- Set as BIOCRO
+    - `met/path` -- Path to file containing meteorological data
+
+### Model specific input files
 
 List of inputs required by model, such as met, etc.
 
-**Model configuration files**
+### Model configuration files
```

Comment:
Would this be the correct place to (eventually, not necessarily in this PR) document the genus-specific parameter files? They're secretly required, just usually looked up from the package. 

My understanding is that `write.configs.BIOCRO` looks for defaults in this order: first any file at a path specified by `settings$pft$constants$file`, next by matching the genus name in datasets exported by the `BioCro` package, last by matching the genus name in `PEcAn.BIOCRO`'s `extdata/defaults` directory.

Reply:
This seems like good information to add to me! I didn't know anything about this, though having this work under the hood is some of why Pecan is useful.